test/utils/configReader.js

```diff
@@ -39,7 +39,13 @@ ConfigReader.prototype = {
   gas: 7500000,
   ZERO_BYTES: '0x0000000000000000000000000000000000000000000000000000000000000000',
   NULL_ADDRESS: '0x0000000000000000000000000000000000000000',
-  tokenHolderBalance: '1111'
+  tokenHolderBalance: '1111',
+  baseCurrencyCode: 'OST',
+  conversionRate: 2,
+  conversionRateDecimals: 18,
+  requiredPriceOracleDecimals: 18,
+  sessionKeysSpendingLimits: [1000000],
```

Comment:
I believe we should not have arrays in configReader.
Instead it should be a value.
Suggestion :- 
`sessionKeysSpendingLimits: [1000000]` => `sessionKeysSpendingLimits: 1000000`
Same comment for `sessionKeysExpirationHeights`.